DBM-Core/DBM-Core.lua

```diff
@@ -4178,7 +4178,7 @@ do
 			private.chatBubblesDisabled = true
 		end
 		if activated then
-			AddMsg(L.OVERRIDE_ACTIVATED)
+			DBM:AddMsg(L.OVERRIDE_ACTIVATED)
```

Comment:
@emmericp is this change needed? the function is cached
 https://github.com/DeadlyBossMods/DBM-Unified/blob/a1d0466ea4f80c54eac0b3e1351756948dd5a17d/DBM-Core/DBM-Core.lua#L7023

Reply:
the self param was missing, it's either this or `AddMsg(DBM, L....)`

Reply:
understood. but then it needs to be AddMsg(DBM, "Foo"), the old code just printed nil.